tests/common/test_cgroupconfigurator.py

```diff
@@ -656,41 +637,69 @@ def get_completed_process():
                     completed = random.randint(1000, 10000)
                 return completed
 
-            def get_telemetry_event_messages():
-                return [kwargs["message"] for (_, kwargs) in add_event_patcher.call_args_list if "The agent's cgroup includes unexpected processes" in kwargs["message"]]
-
             agent_processes = [os.getppid(), os.getpid()] + agent_command_processes + [start_extension.systemd_run_pid]
             other_processes = [1, get_completed_process()] + extension_processes
 
             with patch("azurelinuxagent.common.cgroupconfigurator.CGroupsApi.get_processes_in_cgroup", return_value=agent_processes + other_processes):
-                with patch("azurelinuxagent.common.cgroupconfigurator.add_event") as add_event_patcher:
-                    cgroup_configurator = CGroupConfigurator.get_instance()
+                with self.assertRaises(CGroupsException) as context_manager:
```

Comment:
Now _check_processes_in_agent_cgroup raises an exception and the caller sends the telemetry